### PR TITLE
[WIP] Salmon update 0.14.1

### PR DESCRIPTION
**Pull request by astrovsky01**
For commit history, see https://github.com/galaxyproject/tools-iuc/pull/2501/

### CODE REVIEW

File: tools/salmon/salmonquant.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<tool id="salmon" name="Salmon quant" version="@VERSION@">
```
Comment (bgruening) [inline tools/salmon/salmonquant.xml]: this tool should overwrite this here: https://github.com/astrovsky01/galaxytools/blob/master/tools/salmon/salmon.xml

Comment (astrovsky01) [reply tools/salmon/salmonquant.xml]: Removed salmon.xml

**Comment by bgruening** [comment]
Its finally green, that is a good sign :)

File: tools/salmon/salmonquant.xml [hunk 2]
```diff
@@ -152,8 +152,78 @@
...
+                    <param name="TranscriptSource" value="history"/>
+                    <section name="s_index">
+                        <param name="fasta" value="transcripts.fasta"/>
+                    </section>
+                </conditional>
+                <section name="input">
+                    <param name="single_or_paired.single_or_paired_opts" value="paired" />
+                    <param name="single_or_paired.input_mate1" value="fastqs/reads_1.fastq" />
+                    <param name="single_or_paired.input_mate2" value="fastqs/reads_2.fastq" />
+                    <param name="single_or_paired.oreintation" value="U"/>
```
Comment (bgruening) [inline tools/salmon/salmonquant.xml]:
```suggestion
                    <param name="single_or_paired.orientation" value="U"/>
```

File: tools/salmon/salmonquant.xml [hunk 3]
```diff
@@ -152,8 +152,78 @@
...
```
Comment (bgruening) [inline tools/salmon/salmonquant.xml]:
```suppgestion
                    <param name="single_or_paired.orientation" value="U"/>
```

Comment (astrovsky01) [reply tools/salmon/salmonquant.xml]: The postSample.bam is showing up as the proper size with planemo test --no_cleanup, so it's pulling incorrectly. And I'll add in the loc file tests.

**Comment by mvdbeek** [comment]
Also make sure to either squash on merge or combine ce1556254d813a68fe7557ddb8900555a1677718 and 96e32add68c50347feb86e39b8564514e9a67c58, otherwise the test data stays in the commit history

File: tools/salmon/salmonquant.xml (hunk 3 above)
Comment (astrovsky01) [reply tools/salmon/salmonquant.xml]: Also fixed to strandedness

**Comment by bgruening** [comment]
@astrovsky01 not sure why this is closed? @astrovsky01 I updated your branch. I found a few mistakes but I think the overall error is inside Galaxy. I could not get the loc file test working with latest Galaxy dev.

Can you try your updated branch with 19.05?

**Pull request closed without merging**